### ISSUE TITLE
"Visibility"/"Protected" value is not shown in Firefox

### ISSUE THREAD

**Issue by Mihaiii** [Starcounter/Content#83]
### Screenshot
- Screenshot 1: 
<img width="561" alt="firefox2" src="https://user-images.githubusercontent.com/8951553/28494231-4d0ba8ec-6f28-11e7-8890-0cd1d9871abe.png">

- Screenshot 2:
<img width="518" alt="firefox1" src="https://user-images.githubusercontent.com/8951553/28494228-4224d69c-6f28-11e7-9d09-b368245ce59b.png">

**Comment by Mihaiii** [comment]
same as here: https://github.com/Juicy/imported-template/issues/21

**Comment by Mihaiii** [comment]
After spending some time on this, I got to the conclusion that the pattern we're using now with attaching methods to the template might just not be suitable for Firefox and Edge.

I am talking about stuff like this:

```
            template.deleteItem = function (e) {
                itemToDelete = e.target.item;
                this.$.dialogContainer.open();
            }
```

But then I thought that I might be wrong because this pattern applies in other Starcounter apps too and I should just get the solution from there.

Turns out that in other apps also this doesn't work, but people don't seem to notice (just open some solution, search for ")}}" and go to that page in Edge or Firefox to see if that function is even called).

Well, anyway, since it's the standard pattern, there has to be a solution that was implemented somewhere else, right? Right?

I posted a link in the above comment because @miyconst actually signaled this before (I'm not sure about the "script execution order" part, but the problem is clearly the same). I had a look at the [history of the file](https://github.com/Juicy/imported-template/commit/33789924ff7b3f4fe0322d6b8698ed665a39dcd6#diff-e02e5f9d0dacac8339826683ff5d4e57R17).
That's not really a solution and, besides, I'm having the function inside a `dom-repeat`.

**Any point to an example of a fix for this in other app would be appreciated.**
